scripts/src/logseq/tasks/malli.clj

```diff
@@ -0,0 +1,19 @@
+(ns logseq.tasks.malli
```

Comment:
Like for spec, demonstrating that we can reuse malli schemas from bb (and later nbb if we'd like)